huxley/www/js/components/RegistrationView.js

```diff
@@ -112,301 +116,187 @@ var RegistrationView = React.createClass({
 
   render: function() {
     var conference = this.context.conference;
+
+    //I feel like this (And a lot of others in this section) could 
+    //Be reduced with some kind of mapping,
+    //But there's no obvious way to map over Objects in JS so :/
+    var accountInformationErrors = {
```

Comment:
If you want to map over objects, there are ways to make functions that can do it. But I don't see the explicit need to separate these into variables, the information should be tracked as part of the state